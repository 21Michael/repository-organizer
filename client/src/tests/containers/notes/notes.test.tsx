import React from "react";
import { render, cleanup } from "@testing-library/react";
import Notes from "../../../containers/notes/notes";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("Notes: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("notes", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Notes />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
