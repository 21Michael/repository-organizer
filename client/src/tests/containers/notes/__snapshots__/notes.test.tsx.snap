// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes:  notes 1`] = `
<div>
  <main
    class="main"
  >
    <div
      class="alert__wrapper"
    >
      <div
        class="alert"
      >
        <p>
          Please login for making notes
        </p>
      </div>
    </div>
  </main>
</div>
`;
